frontend/src/pages/order/components/NavBar.tsx

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components'
+import { T } from '../../../reusable-ui/design-system'
+import { CrazeeBurgerLogo } from '../../login/components/CrazeeBurgerLogo'
+import { RightSideNavbar } from './RightSideNavbar'
+
+export function NavBar(props: { firstName: any }) {
+  function reloadPage() {
+    return () => window.location.reload()
```

Comment:
```suggestion
    window.location.reload()
```

cela suffit largement